src/lib.rs

```diff
@@ -20,7 +20,22 @@ fn default_result<T>() -> Result<T, Error> {
     ))
 }
 
-/// Return the MTU of the interface that is used to reach the given remote socket address.
+/// Return the maximum transmission unit (MTU) of the local network interface towards the
+/// destination [`SocketAddr`] given in `remote`.
+///
+/// The returned MTU may exceed the maximum IP packet size of 65,535 bytes on some
+/// platforms for some remote destinations. (For example, loopback destinations on
+/// Windows.)
+///
+/// # Examples
+///
+/// ```
+/// # use std::net::ToSocketAddrs;
+/// # use mtu::get_interface_mtu;
```

Comment:
@mxinden should I just remove these lines?

Reply:
Well, at least the `ToSocketAddrs` one.

Reply:
Adding `# <some rust code>` simply hides `<some rust code>` when rendering the docs. They are still necessary though.

Reply:
https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html#hiding-portions-of-the-example